miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/errors/MiniAppPointsError.kt

```diff
@@ -0,0 +1,16 @@
+package com.rakuten.tech.mobile.miniapp.errors
+
+/**
+ * A class to provide the custom errors specific for user's points.
+ */
+class MiniAppPointsError(val type: String? = null, val message: String? = null) :
+    MiniAppBridgeError(type, message) {
+
+    companion object {
+        /**
+         *  send custom error message from host app.
+         *  @property message error message send to mini app.
+         */
+        fun custom(message: String) = MiniAppPointsError(message = message)
+    }
+}
```

Comment:
Do we need this class? As we don't have any specific type for this error and only message, so may be we can use `MiniAppBridgeError` for this as both looks identical..but if we include specific types in future we can implement like this, what do you think?

Reply:
Yeah, we can use `MiniAppPointsError` to denote the feature specific error class, also we can use the `custom` function to pass the message only.